sdc-plugin/tests/get_clocks/get_clocks.golden.txt

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
This first line is definitely wrong?  `get_clocks` should return something???

Reply:
Fixed.